nixos/modules/services/misc/synergy.nix

```diff
@@ -42,6 +42,13 @@ in
           type = types.bool;
           description = "Whether the Synergy client should be started automatically.";
         };
+        tls = {
+          enable = mkOption {
+            type = types.bool;
+            default = false;
```

Comment:
Wouldn't we want to activate that by default?

Reply:
Well, iirc the server side of the encryption requires a license key to turn on.  I went with false as default to keep from changing the out-of-the-box behavior.  I'm open to either approach.